cdap-ui/app/tracker/services/my-jump-factory.js

```diff
@@ -26,7 +26,7 @@ function myJumpFactory($state, myTrackerApi, $rootScope) {
   let corePluginsArtifacts = {
     name: 'core-plugins',
     scope: 'SYSTEM',
-    version: '1.6.0'
+    version: '1.6.1'
```

Comment:
on develop, I think this is 1.7.0-SNAPSHOT.
https://github.com/caskdata/hydrator-plugins/blob/develop/core-plugins/pom.xml#L26